src/test/java/com/stripe/functional/InvoiceTest.java

```diff
@@ -256,4 +258,22 @@ public void testInvoiceMetadata() throws StripeException {
         params.put("customer", invItem.getCustomer());
         testMetadata(Invoice.create(params));
     }
+
+    @Test
+    public void testSendInvoice() throws StripeException {
+        InvoiceItem invItem = InvoiceItem.create(getInvoiceItemParams());
+        Map<String,Object> params = new HashMap<String,Object>();
+        Long dueDate = (System.currentTimeMillis() / 1000) + 600;
+        params.put("customer", invItem.getCustomer());
+        params.put("billing", "send_invoice");
+        params.put("due_date", dueDate);
+        Invoice invoice = Invoice.create(params);
+        assertEquals(invoice.getBilling(), "send_invoice");
+        assertEquals(invoice.getDueDate(), dueDate);
```

Comment:
Assertion order reversed on both of these assertions. Should be `assertEquals(expect, actual)` http://junit.sourceforge.net/javadoc/org/junit/Assert.html#assertEquals(java.lang.Object, java.lang.Object)